assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/codenvy/che.properties

```diff
@@ -101,10 +104,10 @@ che.workspace.hosts=NULL
 # This is the API endpoint of the workspace master running within the core Che server.
 # This tells the workspace agent how to connect back to the Che server.
 # che-host is a hostname entry added to /etc/hosts of the workspace by the Che server.
-che.workspace.che_server_endpoint=http://che-host:${CHE_PORT}/wsmaster/api
```

Comment:
Please take a look usage of [CHE_WORKSPACE_CHE__SERVER__ENDPOINT](https://github.com/eclipse/che/blob/c4346a9663f06b269dedef82b52a5a4c8ed80bfa/dockerfiles/init/manifests/che.env#L186) env variable